spec/take_order_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'take_order'
+
+describe Checkout do
+
+  let(:checkout){ Checkout.new }
+  let(:checkout2){ double :checkout, :pay_money => @money != @total }
+
+  it 'should tell the customter that their food will be delivered with an hour' do
+    expect(checkout.delivery_time).to eq (Time.new + 3600)
+  end #when this is tested it fails as the time is less than a millisecond apart
+
+  it 'should raise an error if the money doesn\'t match the total' do
+    expect(checkout2.pay_money).to raise_error(RuntimeError, "You have entered an incorrect amount, please reenter")
+  end
+
```

Comment:
No tests for the text message - you should be stubbing out the Twilio client to ensure that the correct messages are being sent, but the actual API will not be contacted every time you run your tests.